gauntlet/src/lib.rs

```diff
@@ -101,12 +101,17 @@ pub struct Args {
     #[arg(short, long)]
     pub quiet: bool,
 
-    /// Overwrites the configuration file with new expected diagnostics and the
-    /// latest commit hashes. This will create temporary shallow clones of every
-    /// test repository. Normally, there is only one repository on disk at a
-    /// time. The difference in disk space usage should be negligible.
+    /// Overwrites the configuration file with new expected diagnostics.
+    ///
+    /// Mutually exclusive with `--update`.
     #[arg(long)]
-    pub refresh: bool,
+    pub bless: bool,
```

Comment:
You need to put these into `clap` argument groups so that they will be shown as mutually exclusive and checked at argument parse time.

Reply:
E.g., https://docs.rs/clap/latest/clap/builder/struct.ArgGroup.html. There is a way to do that with derive as well but you'll need to search for it.